packages/core/src/index.ts

```diff
@@ -202,6 +202,8 @@ class Hopr<Chain extends HoprCoreConnector> extends EventEmitter {
         transport: {
           HoprConnect: {
             bootstrapServers: options.bootstrapServers
+            // Testing
```

Comment:
clean up?

Reply:
This is the setting to enable simulated NAT testing.

Reply:
if it's commented out, we should remove it.

Reply:
This is intentionally not removed to make simulated NAT testing easier as we only need to uncomment this line